UnityProject/Assets/Scripts/Items/Dice/RollDie.cs

```diff
@@ -69,7 +69,11 @@ private void OnDisable()
 	{
 		netTransform.OnThrowStart.RemoveListener(ThrowStart);
 		netTransform.OnThrowEnd.RemoveListener(ThrowEnd);
-		cookable.OnCooked.RemoveListener(Cook);
+
+		if (cookable != null)
+		{
+			cookable.OnCooked.RemoveListener(Cook);
```

Comment:
UnityEvents are cool when you know you have many listeners, for 1 listener it is better to use native delegates.

Here is some data https://jacksondunstan.com/articles/3335

Reply:
but if you are exposing the event to inspector, then Unityevent is the way to go.